tensorflow/tensorboard
# Upgrade pinned version of black.

This upgrades the pinned version of black to 22.6.0.

I encountered this error when running `black .` on my tensorboard repo:

```
Traceback (most recent call last):
  File "/usr/local/google/home/bdubois/virtualenv/tensorboard-git/bin/black", line 8, in <module>
    sys.exit(patched_main())
  File "/usr/local/google/home/bdubois/virtualenv/tensorboard-git/lib/python3.10/site-packages/black/__init__.py", line 1322, in patched_main
    patch_click()
  File "/usr/local/google/home/bdubois/virtualenv/tensorboard-git/lib/python3.10/site-packages/black/__init__.py", line 1308, in patch_click
    from click import _unicodefun
ImportError: cannot import name '_unicodefun' from 'click' (/usr/local/google/home/bdubois/virtualenv/tensorboard-git/lib/python3.10/site-packages/click/__init__.py)
```

According to https://stackoverflow.com/questions/71673404/importerror-cannot-import-name-unicodefun-from-click thiw as fixed in 22.3.0.

I choose to upgrade to the slightly more recent 22.6.0.

To test:
```
$ pip install -r tensorboard/pip_package/requirements.txt -r tensorboard/pip_package/requirements_dev.txt
$ black .
```
Examine diff using `git diff` to check if formatting changes seem reasonable. Ensure python tests pass and tensorboard builds and runs.
